src/samples/flow/iio/sol-flow-intel-minnow-max-linux_gt_3_17.json

```diff
@@ -64,6 +64,14 @@
            "buffer_size": "-1"
      },
      "type": "iio/thermometer"
+  },
+  {
+     "name": "MyProximity",
+     "options": {
+           "iio_device": "apds9960 create,i2c,platform/80860F41:05,0x39,apds9960 0",
+           "buffer_size": "-1"
```

Comment:
I got your point. Curently the “buffer_size=0” will use the default buffer size, it does not fallback to “normal reading” if the driver does not support buffering. We need to enhance the logic, to check if it supports buffering when “buffer_size > 0”